### PR TITLE
Marketplace Recall + some other changes

### PR DESCRIPTION
**Pull request by LtRipley36706**

### CODE REVIEW

File: Source/ACE.DatLoader.Tests/DatTests.cs [hunk 1]
```diff
@@ -24,5 +25,63 @@ public void LoadPortalDat_NoExceptions()
             int count = dat.AllFiles.Count();
             Assert.AreEqual(79694, count);
         }
+
+        [TestMethod]
```
Comment (Mogwai-AC) [inline Source/ACE.DatLoader.Tests/DatTests.cs]: These take an hour+ to run and leave 800,000 files on your computer. Please comment out the TestMethod attribute at a minimum - i want to be able to run tests safely.

Comment (LtRipley36706) [inline Source/ACE.DatLoader.Tests/DatTests.cs]: I believe I've resolve this issue, do the below changes make sense?

Comment (Mogwai-AC) [inline Source/ACE.DatLoader.Tests/DatTests.cs]: i was going for "// [TestMethod]", but yes

**Comment by LtRipley36706** [comment]
Working on a few changes

**Comment by LtRipley36706** [comment]
moving stuff around and redoing PR

**Pull request closed without merging**